test/unit/server_test.js

```diff
@@ -1117,5 +1117,556 @@ describe("server.js tests", function () {
           })
       });
     });
+
+
+    describe("AssetsCallBuilder", function() {
+      let assetsResponse = {
+        "_links": {
+          "self": {
+            "href": "http://horizon-testnet.stellar.org/assets?order=asc\u0026limit=10\u0026cursor="
+          },
+          "next": {
+            "href": "http://horizon-testnet.stellar.org/assets?order=asc\u0026limit=10\u0026cursor=00838f_GD4SAUKGB6GE2Q25H2CZMZ3BSP5CVYIY2LQYJDCFNNICR473AVL7IYH5_credit_alphanum12"
+          },
+          "prev": {
+            "href": "http://horizon-testnet.stellar.org/assets?order=desc\u0026limit=10\u0026cursor=004d40_GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN_credit_alphanum12"
+          }
+        },
+        "_embedded": {
+          "records": [
+            {
+              "_links": {
+                "toml": {
+                  "href": ""
+                }
+              },
+              "asset_type": "credit_alphanum12",
+              "asset_code": "004d40",
+              "asset_issuer": "GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN",
+              "paging_token": "004d40_GCOGPF7IRVXUCJZAQWXVFQEE4HAOCTDGZI2QZSMKLM5BTTGRLY6GDOJN_credit_alphanum12",
+              "amount": "757.0000000",
+              "num_accounts": 18,
+              "flags": {
+                "auth_required": false,
+                "auth_revocable": false
+              }
+            },
+            {
```

Comment:
Is there any value in having more than a single record in these mock responses? 
Not that it matters, but this file is turning into a monster so if not needed, let's keep these responses small.